tools/chapel-py/setup.py

```diff
@@ -69,6 +69,10 @@
     chpl_lib_path,
 ]
 
+if str(chpl_variables.get("CHPL_SANITIZE")) == "address":
+    CXXFLAGS += ["-fsanitize=address"]
+    LDFLAGS += ["-fsanitize=address"]
+
```

Comment:
Can you also add some kind of checking here to prevent/warn users building this config on MacOS and expecting it to work?